app/src/debugUnitTests/java/com/artemzin/qualitymatters/developer_settings/DeveloperSettingsModelImplTest.java

```diff
@@ -33,8 +35,8 @@ public void beforeEachTest() {
                 developerSettings,
                 mock(OkHttpClient.class),
                 new HttpLoggingInterceptor(),
-                mock(LeakCanaryProxy.class)
-        );
+                mock(LeakCanaryProxy.class),
+                new Paperwork(mock(QualityMattersApp.class)));
```

Comment:
Well, because originally the tests here wanted to see real results returned. But you are right, now it should be mocked and only the interaction tested.